recipes/emacs/build.sh

```diff
@@ -0,0 +1,7 @@
+if [ "$(uname)" == "Darwin" ]; then
+    ./configure  --prefix=$PREFIX --without-x
+else
+    ./configure  --prefix=$PREFIX --x-includes=$PREFIX/include --x-libraries=$PREFIX/lib
+fi
+
+make -j4  && make -j4  install
```

Comment:
Please don't hard-code the parallel build.  We're working on a better way (allowing pass-through of the CPU_COUNT variable to override autodetection), but it's not ready yet.